geomstats/special_orthogonal_group.py

```diff
@@ -112,29 +112,49 @@ class SpecialOrthogonalGroup(LieGroup, EmbeddedManifold):
     i.e. the Lie group of rotations.
     """
 
-    def __init__(self, n):
+    def __init__(self, n, point_representation=None):
```

Comment:
would just "representation" instead of point_representation work?

if not would point_type work?

just trying to make things shorter.

Reply:
Ah yes, good idea.